src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -39,7 +39,10 @@ class UserService(
       is ClientResult.Failure -> staffUserDetailsResponse.throwException()
     }
 
-    if (deliusUser.email !== user.email || deliusUser.telephoneNumber !== user.telephoneNumber) {
+    val name = "${deliusUser.staff.forenames} ${deliusUser.staff.surname}"
+
+    if (deliusUser.email !== user.email || deliusUser.telephoneNumber !== user.telephoneNumber || name != user.name) {
```

Comment:
Would the purpose of this perhaps be clearer if this became a predicate function, e.g. `hasUserChanged()`? I think that's what this is doing: updating our user record if we detect a change at Delius?

Reply:
Good idea! I've made that change 👍